upgrade/Stage_0_Prerequisites.md

```diff
@@ -277,9 +278,9 @@ In most cases, administrators will be performing a standard upgrade and not a CS
     This will ensure that the CFS configuration layers are applied to perform image customization for the worker NCNs.
     See [Worker Image Customization](../operations/configuration_management/Worker_Image_Customization.md).
 
-1. Prepare the post-boot worker NCN personalizations.
+1. Prepare the post-boot NCN personalizations.
 
-    This will ensure that the appropriate CFS configuration layers are applied when performing post-boot node personalization of the worker NCNs.
+    This will ensure that the appropriate CFS configuration layers are applied when performing post-boot node personalization of the NCNs.
```

Comment:
```suggestion
    This will ensure that the appropriate CFS configuration layers are applied when performing post-boot node personalization of the master, storage, and worker NCNs.
```